stores/healthcareProfessionalsStore.ts

```diff
@@ -92,3 +107,12 @@ mutation Mutation($updateHealthcareProfessionalId: ID!, $input: UpdateHealthcare
   }
 }
 `
+
+const deleteHealthcareProfessionalGqlMutation = gql`
+mutation Mutation($deleteHealthcareProfessionalId: ID!) {
+  deleteHealthcareProfessional(id: $deleteHealthcareProfessionalId) {
+    isSuccessful
```

Comment:
We should use this response

Reply:
It should he used in the caller of the delete action. We should include it in the return type

Reply:
@ermish I commented where I am using it on the line. For clarification 🙇